processors/include/BhToysHistoProcessor.h

```diff
@@ -77,6 +88,9 @@ class BhToysHistoProcessor : public Processor {
         // of the invariant mass distribution. The number of events may be modified
         // by a multiplicative factor.
         int bkg_mult_{1};
+
+        // The factor by which to scale the mass resolution function.
+        double res_scale_{1.56};
```

Comment:
This should probably be 1.00 to be safe.

Reply:
This default was all over the place... Fixed it here too.